gratipay/security/__init__.py

```diff
@@ -41,6 +41,10 @@ def add_headers_to_response(response):
     if 'X-XSS-Protection' not in response.headers:
         response.headers['X-XSS-Protection'] = '1; mode=block'
 
+    # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
+    if 'strict-transport-security' not in response.headers:
+        response.headers['strict-transport-security'] = 'max-age=31536000; includeSubDomains'
```

Comment:
We would need to land https://github.com/gratipay/inside.gratipay.com/issues/943 before rolling this out with `includeSubDomains`.